tests/vmi_multus_test.go

```diff
@@ -797,11 +797,18 @@ var _ = Describe("[Serial]SRIOV", func() {
 			startVmi(vmi)
 			waitVmi(vmi)
 
-			vmi, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
-			Expect(err).NotTo(HaveOccurred())
-
-			interfaceName, err := getInterfaceNameByMAC(vmi, mac)
-			Expect(err).NotTo(HaveOccurred())
+			vmiNamespace := vmi.Namespace
+			vmiName := vmi.Name
+			var interfaceName string
+			Eventually(func() error {
+				var err error
```

Comment:
nit: I think you can remove the var statement and use ":=" instead of "="  on the next line.

Reply:
we cant, because vmi isnt new

Reply:
/lgtm